api/schemas_service.go

```diff
@@ -0,0 +1,38 @@
+//
```

Comment:
Can we rename this file from `api/schemas_service.go` to `api/schemas.go`, we don't have
a standard to prepend `_service.go` to a "service" so, it might be confusing.